apps/api/.env.example

```diff
@@ -0,0 +1,5 @@
+RECAPTCHA_SITE_KEY=
+FIREBASE_PROJECT_ID=
+GOOGLE_APPLICATION_CREDENTIALS=
+FRONTEND_URL=
+ENVIRONMENT=local
```

Comment:
What is the point of specifying ‘local’ in the example?

Reply:
It's just that I think, when someone want to work on the project he would not use something else as environment than the local one 😉